setup.py

```diff
@@ -45,7 +45,7 @@
     'h5py >= 2.7.1',
     'plotly>=4.0.0',
     'joblib >= 0.16.0',
-    'xgboost >= 1.3.3, <1.5.0',
+    'xgboost >= 1.5.1',
```

Comment:
What's the reason for updating the minimum?  I thought the motivation for switching to JSON was so that we wouldn't have to worry about keeping track of xgboost versions.  1.3.3 seems to work for me locally with the JSON model, so maybe we don't actually need to change the version to 1.5.1 here and in the two requirements files?  So long as it's not a headache for us it's nice to not be overly restrictive on dependency version ranges, so keeping 1.3.3 would be preferable if it doesn't break anything.